src/ceph.in

```diff
@@ -552,8 +552,8 @@ def ping_monitor(cluster_handle, name, timeout):
         return 1
 
     mon_id = name[len('mon.'):]
+    run_in_thread(cluster_handle.connect, timeout=timeout)
```

Comment:
cluster_handle.ping_monitor doesn't work if the cluster is not connected, which is why I found this bug.  The error message is "Error connecting to cluster: ObjectNotFound".  I'll investigate why, exactly.